### PR TITLE
Bugfix: relax the requirements

### LINKED ISSUE #143: [Query] Why is torch 1.5.0 imposed?
In `requirements.txt` the first line `torch>=1.2.0,!=1.5.0` uninstalls existing pytorch version and re-installs this particular version. This could end up in breaking the entire existing pipeline in some case. 
Any particular reason why is this being imposed? 
Is `torch>=1.2.0` already being tried and the results weren't good enough?

### PR DESCRIPTION
**Pull request by denproc**
Closes #143 
The problem was related to the upgrade of the environment with `torch` incompatible with CUDA. 
## Proposed Changes

  - made requirements on `pytorch` and `torchvision` less strict
  - added warning to the BRISQUE score that backprop is not available using `torch==1.5.0'
  - added notes to the docstring and readme

### CODE REVIEW

**Comment by denproc** [comment]
Ready to merge after #153

**Comment by denproc** [comment]
Let me look into it a bit more after @snk4tr comment

**Comment by denproc** [comment]
@snk4tr @zakajd the proposed approach works, ready for review.

As I suggested, requirement `torch>=1.2` covers the tags used to specify the cuda driver version.

Additional context just in case:
```python
[In]: pip list
[Out]: torch                  1.5.0+cu101 
[Out]: torchvision        0.6.0+cu101 

[In]: python setup.py install
[In]: pip list
[Out]: piq                      0.5.1    
[Out]: torch                  1.5.0+cu101 
[Out]: torchvision        0.6.0+cu101 
```

File: README.md [hunk 1]
```diff
@@ -51,7 +51,9 @@ prediction = torch.rand(3, 3, 256, 256)
 brisque_index: torch.Tensor = brisque(prediction, data_range=1.)
 ```
 
-In order to use BRISQUE as a loss function, use corresponding PyTorch module:
+In order to use BRISQUE as a loss function, use corresponding PyTorch module. 
+
+Note: the back propagation is not available using `torch==1.5.0`. Update the environment with latest `torch` and `torchvision`.
```
Comment (snk4tr) [inline README.md:56]: Please also add `warnings.warn` at the start of execution of BRISQUE computations if `torch.__version__ == 1.5.0`
